SPECS/qemu-kvm/qemu-kvm.spec

```diff
@@ -138,7 +140,50 @@ ln -sv qemu-system-`uname -m` %{buildroot}%{_bindir}/qemu
 chmod 755 %{buildroot}%{_bindir}/qemu
 
 %check
-# Deliberately empty
+testsPassed=true
+make check-unit
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    testsPassed=false
+fi
+make check-qtest
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    testsPassed=false
+fi
+make check-speed
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    testsPassed=false
+fi
+make check-qapi-schema
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    testsPassed=false
+fi
+make check-block
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    testsPassed=false
+fi
+make check-tcg
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    testsPassed=false
+fi
+make check-softfloat
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    testsPassed=false
+fi
+make check-acceptance
+retVal=$?
+if [ $retVal -ne 0 ]; then
+    testsPassed=false
+fi
+if [ "$testsPassed" = false ] ; then
+    echo 'Some tests failed. Check log for further details'
```

Comment:
Can this `exit 1` so the %check section returns a failure overall?
I think an exit here would allow the rest of the RPM build to continue on a check fail, but you should double check.

Reply:
Done, added (exit 1)